core/src/tile/tileWorker.cpp

```diff
@@ -94,10 +97,10 @@ void TileWorker::run() {
 
             // float loadTime = (float(clock() - begin) / CLOCKS_PER_SEC) * 1000;
             // LOG("loadTime %s - %f", task->tile->getID().toString().c_str(), loadTime);
-
-            m_tileManager.tileProcessed(std::move(task));
```

Comment:
So in a case when a tile's `tileData` is not processed and is marked `ready` by `TileManager::tileProcessed`, line: 227 in `tileManager.cpp` will clear the corresponding proxy tiles, even though the actual ready tile is not processed!! :(